### DISCUSSION TITLE
以vue2项目为主应用，vue3官网 快速上手cdn demo为子应用，是什么问题？

### DISCUSSION THREAD

**Discussion by longshenzhu** [jd-opensource/micro-app#1138] [General]
![demo_tab](https://github.com/micro-zoe/micro-app/assets/3724569/5eb8cc21-8d02-4f3e-8f4c-4612167b6f1a)

**Comment by bailicangdu** [comment]
由于cdn js中var Vue无法泄漏为全局变量导致，推荐其它创建案例的方式
